components/automate-ui/src/app/modules/infra-proxy/create-chef-server-modal/create-chef-server-modal.component.html

```diff
@@ -10,7 +10,7 @@ <h2 slot="title">Add Chef Server </h2>
           </label>
           <chef-error
           *ngIf="(createForm.get('name').hasError('required') || createForm.get('name').hasError('pattern')) && createForm.get('name').dirty">
-          Display Name is required.
+          Name is required.
```

Comment:
Author note: consistency -- error message must match actual displayed field name (which occurs a few lines higher in the file).